### PR TITLE
add lint

### PR DESCRIPTION
**Pull request by cpanato**
Adding inital version of the makefile and add at least lint to check prs

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+GO ?= $(shell command -v go 2> /dev/null)
+DEP ?= $(shell command -v dep 2> /dev/null)
+
+PACKAGES=$(shell go list ./...)
+
+## Checks the code style, tests, builds and bundles the plugin.
+all: check-style
```
Comment (hmhealey) [inline Makefile]: This needs a `.PHONY`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (hmhealey) [inline Makefile]: There's an extra blank line here. Maybe we need a lint for Makefiles 😛

File: Makefile (hunk 1 above)
Comment (cpanato) [reply Makefile]: done

File: Makefile (hunk 2 above)
Comment (cpanato) [reply Makefile]: done

**Pull request merged**